hr_employee_exemption/hr_employee_exemption.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Savoir-faire Linux. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+
+
+class hr_employee_exemption(orm.Model):
+    _name = 'hr.employee.exemption'
+    _description = 'Employee Income Tax Exemption'
+
+    _columns = {
+        'employee_id': fields.many2one(
+            'hr.employee',
+            'Employee',
+            required=True,
+            ondelete='cascade',
+        ),
+        'exemption_id': fields.many2one(
```

Comment:
@feketemihai, @pedrobaeza  I agree about linking the exemption to the salary rule. For the employee benefit module, what about that split:
hr_employee_benefit (depends on hr_payroll)

hr_employee_benefit_on_job (depends on  hr_employee_benefit, hr_contract_multi_jobs and hr_worked_days_activity)
     add employee benefits based on each worked hours of the employee

hr_employee_benefit_exemption (depends on hr_employee_benefit, hr_employee_exemption)
     add exemptions on benefits

hr_employee_benefit_percent_gross (depends on hr_employee_benefit)
    add benefits based on percent of the gross slary

What do you think ?